debian/homestead.postinst

```diff
@@ -89,13 +89,6 @@ case "$1" in
 
         service clearwater-infrastructure restart
 
-        # Stop the cluster manager, so that it is restarted by Monit and picks
-        # up the new Cassandra scaling plugin. We check whether the process is 
-        # installed to avoid warning logs when etcd isn't being used
-        if [ -x "/etc/init.d/clearwater-cluster-manager" ]; then
-          service clearwater-cluster-manager stop || /bin/true
-        fi  
-
```

Comment:
Commit https://github.com/Metaswitch/homestead/commit/be89ba195b8ade458157bc9e58edd7a6c6094648 moved that plugin out but didn't delete this code.